shared/AppInsightsCore/Tests/Unit/src/ApplicationInsightsCore.Tests.ts

```diff
@@ -1908,6 +1908,670 @@ export class ApplicationInsightsCoreTests extends AITestClass {
             }
         });
 
+        this.testCase({
+            name: "FieldRedaction: should redact basic auth credentials from URL",
+            test: () => {
+                let config = {redactUrls: false} as IConfiguration;
```

Comment:
This still result in the if being false... And therefore lines 1918..1919 won't get executed...

If you drop the If check (line 1917) and then change 1919 to check that the `redactedLocation` is still the same as the passed in url would be a better test (assuming I'm understanding what this is trying to test)